components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1951,7 +1951,6 @@ type PortConfig struct {
 	Visibility  string  `json:"visibility,omitempty"`
 	Description string  `json:"description,omitempty"`
 	Name        string  `json:"name,omitempty"`
-	Sort        uint32  `json:"sort,omitempty"`
```

Comment:
it should not be here in the first place. cc @geropl @easyCZ for review of webapp owned files

Reply:
Not mentioned [here](https://github.com/gitpod-io/gitpod/blob/5c8f4c1f32ebc2eb799a1ac4aa0b71eaff1c8336/components/gitpod-protocol/src/protocol.ts#L967-L973) at all. :heavy_check_mark: